codecademy/functions.py

```diff
@@ -1,6 +1,7 @@
 # 4. Function ex-3
 def square(n):
     """This function returns the square of a number."""
+    assert type(n) == int or type(n) == long, "{} was not int".format()
```

Comment:
So this line didn't cause issues with Python3 accidentally, since all the calls had argument `int` and second part of assertion statement didn't get evaluated (`or type(n) == long`), if you'd call square with non int type it would raise exception in Python3 since they got rid of long type in python3 - it's all int now.
better do this
```python
# add to the end of the import section of this file
import sys
if sys.version_info > (3,):
    long = int
```
then optionally improve your assert statement with
```python
assert isinstance(n, (int, long)),  "{} was not int or long".format()
```